pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -169,6 +188,8 @@ type smcpmutator interface {
 	GetPatches() []jsonpatch.JsonPatchOperation
 	IsOpenShiftRouteEnabled() *bool
 	SetOpenShiftRouteEnabled(bool)
+	IsTracingTypeSpecified() bool
+	SetTracingType(v2.TracerType)
```

Comment:
This function always takes one argument - `TracerTypeNone` - so I would change its signature to:
```suggestion
	DisableTracing()
```